### ISSUE TITLE
Documentation is unclear about the configuration of Discord channels

### ISSUE THREAD

**Issue by cahe7cb** [tsoding/HyperNerd#726]
By the [example configuration file](https://github.com/tsoding/HyperNerd#example-of-a-secretini-file) and the lack of plural in the [Discord configuration](https://github.com/tsoding/HyperNerd#discord-config-section) section is unclear that the `channel` field should be a list of channels, that is, surrounded by `[]` (square brackets).

Which is aggravated by the fact that the following (mis)configuration:
```
[discord]
authToken = token
guildId = 123
channel = 123
```
will result in the following output when starting the program:
```
HyperNerd: user error (In file 'secret.ini': channel is not a number)
```

**Comment by rexim** [comment]
@cahe7cb thank you for filing the issue! Should be fixed in 6a02b87cffa174d6c81daaa412271d08141fd0c7 Please close the issue if the documentation is clear now.

**Comment by cahe7cb** [comment]
The documentation is much clearer now as well as the runtime error message! And I can run just fine with the new changes.

Thanks for the quick response. I will close the issue.
